llvm/utils/lit/tests/Inputs/shtest-cat/cat.txt

```diff
@@ -0,0 +1,14 @@
+## Tests
```

Comment:
Please either describe what you're testing, or remove the top comment + newline.

Reply:
done